x/concentrated-liquidity/query_test.go

```diff
@@ -871,3 +874,153 @@ func (s *KeeperTestSuite) TestGetNumNextInitializedTicks() {
 		})
 	}
 }
+
+func (s *KeeperTestSuite) TestGetTickRangeUnderlyingAssets() {
+	// Init suite
+	s.SetupTest()
+
+	// Fund Account
+	fundCoins := sdk.NewCoins(sdk.NewCoin("atom", osmomath.NewInt(1000000000000000000)), sdk.NewCoin("uosmo", osmomath.NewInt(1000000000000000000)))
+	s.FundAcc(s.TestAccs[0], fundCoins)
+
+	// Create a default CL pool
+	s.PrepareCustomConcentratedPool(s.TestAccs[0], "atom", "uosmo", DefaultTickSpacing, osmomath.ZeroDec())
+
+	// Create a full-range position
+	data0, err := s.App.ConcentratedLiquidityKeeper.CreatePosition(
+		s.Ctx,
+		1,
```

Comment:
nit: just a personal pedantic suggestion: I think it would be better to dynamically get pool's id from returned by `PrepareCustomConcentratedPool` pool extension